posthog/migrations/0478_delete_old_alert_models.py

```diff
@@ -0,0 +1,24 @@
+# Generated by Django 4.2.15 on 2024-09-16 14:05
+
+from django.db import migrations
+
+
+class Migration(migrations.Migration):
+    dependencies = [
+        ("posthog", "0477_datawarehouse_salesforce_order"),
+    ]
+
+    operations = [
+        migrations.DeleteModel(
+            name="Alert",
+        ),
+        migrations.DeleteModel(
```

Comment:
Why are you deleting these models (except Alert)? I see they're recreated below, but should we keep and modify them in the migration?

Reply:
Thought since no ones using them, just easier to start from a blank slate without having to add defaults/nullables?